.env.template

```diff
@@ -0,0 +1,3 @@
+MONGODB_URI=mongodb+srv://app_admin:admin@cluster0.torqrki.mongodb.net/?retryWrites=true&w=majority
+NEXTAUTH_URL=http://localhost:3000/
+NEXTAUTH_SECRET=encryptionkey
```

Comment:
Do not push .env files on remote branch.

Reply:
noted mam